recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+{% set version = "6.0.0" %}  # PEP 386
+{% set source_version = version %}
+{% set base_version = version %}
+{% set number = "4" %}
+{% set cuda_enabled = cuda_compiler_version is not undefined and cuda_compiler_version == '11.0' %}
+{% set build_ext_version = "1.0.0" %}
+{% set build_ext = "cuda" if cuda_enabled else "cpu" %}
+{% set build_string = "h{}_{}".format(PKG_HASH, number) %}
+{% set build_string_ext = "{}_{}".format(build_string, build_ext) %}
+{% set py_build_string_ext = "py{}{}_{}".format(CONDA_PY, build_string, build_ext) %}
+{% set py_build_string = "py{}{}".format(CONDA_PY, build_string) %}
+{% set install_base = "opt/heavyai" %}
+{% set arrow_version = "7.*" %}
+{% set arrow_proc_version = "3.*" %}
+{% set pythrift_version = "0.16.*" %}
+{% set thrift_version = "0.16.*" %}
+# heavydb 6.0.0 is not LLVM 12 ready
+{% set llvm_version = "11" %}
+
+package:
+  name: heavydb-ext
+  version: {{ version }}
+
+source:
+  url: https://github.com/heavyai/heavydb/archive/v{{ source_version }}.tar.gz
+  sha256: a36d91227135e9ec8641b5f46e80927aaafce117f3446933dd7f392c9659996e
+  patches:
+    - 0003-Fix-HeavyDBE-build.patch
+
+build:
+  number: {{ number }}
+  skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+
+outputs:
+
+  - name: heavydb-common
+    version: {{ version }}
+    script: build-heavydb.sh  # [linux64]
+    script: build-heavydb-common.bat  # [win]
+    build:
+      string: {{ build_string }}
+      # cpu-only heavydb-common for cuda is required only for
+      # building cuda-enabled heavydb
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0")]
+      skip: true  # [osx]
+      skip: true  # [win and cuda_compiler_version != "None"]
+      run_exports:
+        - {{ pin_subpackage('heavydb-common',  max_pin='x.x.x') }}
+      ignore_run_exports:
+        - arrow-cpp {{ arrow_version }}
+        - blosc
+        - double-conversion
+        - fmt
+        - geos
+        - glog
+        - libarchive
+        - libgdal
+        - libllvm{{ llvm_version }}
+        - libpng
+        - librdkafka
+        - libstdcxx-ng
+        - libthrift
```

Comment:
Are we overriding the run exports with the subpackage's itself? What is this supposed to achieve? I might be missing the details, sorry, but a comment here might be useful.